src/components/Modals/Send/hooks/useFormSend/useFormSend.test.tsx

```diff
@@ -159,8 +159,10 @@ describe.each([
 
     const { result } = renderHook(() => useFormSend())
     await result.current.handleSend(formData)
-    expect(toaster).toHaveBeenCalledWith(expect.objectContaining({ status: 'success' }))
-    expect(sendClose).toHaveBeenCalled()
+    setTimeout(() => {
```

Comment:
`setTimeout` in tests is not the correct approach, please mock and advance timers.

Reply:
Pushed these changes.